lib/config.js

```diff
@@ -18,3 +18,13 @@ config.profile.kms.ipAllowList = [];
 config.profile.kms.defaultKmsModule = 'ssm-v1';
 // ensure default KMS module is overridden in deployments
 config.ensureConfigOverride.fields.push('profile.kms.defaultKmsModule');
+
+config.profile.profileAgent = {
+  // One month time threshold (in milliseconds) for triggering zcap refresh.
+  zcapRefreshThreshold: 30 * 24 * 60 * 60 * 1000,
+  // TTL for profile agent's EDV zcaps: 365 days
+  defaultZcapTtl: 365 * 24 * 60 * 60 * 1000,
+  // TTL for root profile agent's zcap to use profile zcap invocation key:
+  // 1000 years
+  profileCapabilityInvocationKeyZcapTtl: 1000 * 365 * 24 * 60 * 60 * 1000
```

Comment:
This looks like we want to ensure the zcap never expires. Instead of maintaining code for a TTL, can we try something else? When we initially create the zcap, can we use the largest date we can for the expiration? It seems better to create the zcap with a large expiration date vs this approach.

Reply:
I've left `profileCapabilityInvocationKeyZcapTtl` in the config as is but added comment that its value  cannot be higher than 1000 years as it is a hard limit.